### ISSUE TITLE
Views and templates are not translation-friendly

### ISSUE THREAD

**Issue by badrihippo** [sawaliram/sawaliram#153]
Most of the code is hard-coded in English. This code needs to be sifted through, and potential translatable strings be marked as such so they can be picked up by the translation engine for converting to different languages.

This can be done using:

- the `gettext` and `ugettext` functions in views (aliased `_`)
- the `{% trans %} and {% blocktrans %}` templatetags in templates
- some other fancy stuff for JavaScript

All nuances and techniques are outlined [in the official Django docs](https://docs.djangoproject.com/en/3.0/topics/i18n/translation/).

Once this is done, we need to have translators (or automated algorithms) to actually translate the marked text into various languages. That is an independent operation and should perhaps be discussed in a separate issue.
